olp-cpp-sdk-dataservice-read/tests/VersionedLayerClientTest.cpp

```diff
@@ -72,4 +72,18 @@ TEST(VersionedLayerClientTest, GetData) {
   }
 }
 
+TEST(VersionedLayerClientTest, RemoveFromCache) {
+  std::shared_ptr<NetworkMock> network_mock = std::make_shared<NetworkMock>();
```

Comment:
So we need to mock network even for offline tests? Not good😱

Reply:
Oops! Fixed.

Reply:
No worries. I thought that was necessary because of some SDK's internals trying to go online even in such a simple test.